modules/sdk/gradle-plugins-app-javadoc-builder/README.markdown

```diff
@@ -0,0 +1,59 @@
+# App Javadoc Builder Gradle Plugin
+
+The App Javadoc Builder Gradle plugin allows you to generate a combined HTML API
+documentation for an application that spans different subprojects, each one
+representing a different component of the same application.
+
+## Usage
+
+To use the plugin, include it in the build script of the root project:
+
+```gradle
+buildscript {
+	dependencies {
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.app.javadoc.builder", version: "1.0.0"
+	}
+
+	repositories {
+		maven {
+			url "https://cdn.lfrs.sl/repository.liferay.com/nexus/content/groups/public"
+		}
+	}
+}
+
+apply plugin: "com.liferay.app.javadoc.builder"
+```
+
+The App Javadoc Builder plugin automatically applies the [`base`](https://docs.gradle.org/current/userguide/standard_plugins.html#N135C1)
+and the `reporting-base` plugins.
+
+## Project Extension
+
+The App Javadoc Builder plugin exposes the following properties through the
+extension named `appJavadocBuilder`:
+
+Property Name | Type | Default Value | Description
+------------- | ---- | ------------- | -----------
+`copyTags` | `boolean` | `true` | Whether to copy the custom block tags configuration from the subprojects. It sets the Javadoc [`-tag`](http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#tag) argument for the [`appJavadoc`](#appjavadoc) task.
+`doclintDisabled` | `boolean` | `true` on JDK8+, `false` otherwise. | Whether to ignore Javadoc errors. It sets the Javadoc [`-Xdoclint`](docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#BEJEFABE) and [`-quiet`](http://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#CHDGFHAA) arguments for the [`appJavadoc`](#appjavadoc) task.
+`groupNameClosure` | `Closure<String>` | Closure which returns the subproject description, or the name if the description is empty. | The closure invoked in order to get the group heading for a subproject. The given closure is passed a [`Project`](https://docs.gradle.org/current/javadoc/org/gradle/api/Project.html) as its parameter. If `groupPackages` is `false`, this property is not used.
```

Comment:
@Ithildir For the default value of `groupNameClosure`, do you think we can omit restating `Closure`, since it's already stated as the type? Maybe say something like

> The subproject's description. or the subproject's name if the description is empty

Or do you think restating Closure in the default value description is necessary?